client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -93,13 +94,15 @@ def __init__(
         loader_name,
         namespace,
         object_name,
-        item_id
+        item_id,
+        project_name
     ):
         self.representation_id = representation_id
         self.loader_name = loader_name
         self.object_name = object_name
         self.namespace = namespace
         self.item_id = item_id
+        self.project_name = project_name
 
     @classmethod
     def from_container_data(cls, container):
```

Comment:
You should pass in the project name if container data is not enough.

Reply:
I meant to pass it the current project, not to use `None`.

Reply:
I meant to pass current project name as argument to `from_container_data` method.

```python
@classmethod
    def from_container_data(cls, container):
        return cls(
            representation_id=container["representation"],
            loader_name=container["loader"],
            namespace=container["namespace"],
            object_name=container["objectName"],
            item_id=uuid.uuid4().hex,
            project_name=container.get(
                "project_name", get_current_project_name()
            )
        )
```